apps/portal/examples/dotnet-template/content/source/Dockerfile

```
@@ -0,0 +1,17 @@
+COPY DotNetApi.csproj ./
+RUN dotnet restore DotNetApi.csproj
+
+COPY . .
+RUN dotnet publish DotNetApi.csproj -c Release -o /app/publish /p:UseAppHost=false
+
+FROM mcr.microsoft.com/dotnet/aspnet:10.0.5 AS runtime
+WORKDIR /app
+COPY --from=build /app/publish .
+
+ENV ASPNETCORE_URLS=http://+:${{ values.port }}
+EXPOSE ${{ values.port }}
```

Hadolint (2.14.0)

[error] unexpected 'v'
expecting '\', A port optionally followed by the protocol, A range of ports optionally followed by the protocol, a new line followed by the next instruction, or at least one space

(DL1000)